docs/preview/02-Features/powershell/azure-devops.md

```diff
@@ -142,7 +143,7 @@ PS> Set-AzDevOpsArmOutputsToPipelineVariables -ArmOutputsEnvironmentVariableName
 **Azure DevOps Example**
 This function is intended to be used from an Azure DevOps pipeline.
 
-> ⚠ When you are using a Linux agent, you need to pass other environment variables that you want to use as well, because these are not available. To be able to use the `ArmOutputs` environment variable, it must be explicitly added to the environment-variables.
+> ⚠ When you are using a Linux agent (which is recommended), you need to pass other environment variables that you want to use as well, because these are not available. To be able to use the `ArmOutputs` environment variable, it must be explicitly added to the environment-variables.
```

Comment:
```suggestion
> ⚠ When you are using a Linux agent, you need to pass other environment variables that you want to use as well, because these are not available. To be able to use the `ArmOutputs` environment variable, it must be explicitly added to the environment-variables.

> We have seen a much better performance when using Linux agents, and would recommend using Linux agents when possible.
```